src/sentry_plugins/hipchat_ac/static/hipchat_ac/components/settings.jsx

```diff
@@ -58,14 +58,45 @@ class Settings extends plugins.BasePlugin.DefaultSettings {
       }
     });
   }
-
-  renderTenants() {
+  renderLink(url, metadata) {
+    let tenants = this.state.tenants;
+    if (!tenants || !tenants.length) {
+      return (
+        <div>
+          <p>
+            HipChat notifications are managed through HipChat itself.  After you've
```

Comment:
@cameronmcefee might have some thoughts on language, but you use directly twice :)